### PR TITLE
FrontEnd implemented

### PR DESCRIPTION
**Pull request by Zarathustra4**

### CODE REVIEW

File: src/main/java/com/passpnu/passwordmanager/generator/StringPasswordGenerator.java [hunk 1]
```diff
@@ -29,11 +28,8 @@ public String generate(){
         CharacterRule digitRule = new CharacterRule(EnglishCharacterData.Digit);
         digitRule.setNumberOfCharacters(digitCount);
 
-        CharacterRule specialCharRule = new CharacterRule(EnglishCharacterData.Special);
-        specialCharRule.setNumberOfCharacters(specialCharCount);
```
Comment (justme1135) [inline src/main/java/com/passpnu/passwordmanager/generator/StringPasswordGenerator.java:33]: I'm pretty sure adding special characters will be more secure. Why are they deleted?

File: src/main/resources/templates/footer.ftl [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (justme1135) [inline src/main/resources/templates/footer.ftl:1]: maybe add some info about yourself in footer

File: src/main/resources/templates/menu.ftl [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (justme1135) [inline src/main/resources/templates/menu.ftl]: maybe think about more descriptive titles for each page

File: src/main/java/com/passpnu/passwordmanager/generator/StringPasswordGenerator.java (hunk 1 above)
Comment (Zarathustra4) [reply src/main/java/com/passpnu/passwordmanager/generator/StringPasswordGenerator.java:33]: Cause some characters are too special... For example: †1A91Ldr)U, 4Yq6F06i¸³M‗T, -n1VTeVx2, 'r₭fPQ6bZ5H

File: src/main/resources/templates/footer.ftl (hunk 2 above)
Comment (Zarathustra4) [reply src/main/resources/templates/footer.ftl:1]: fixed

File: src/main/resources/templates/menu.ftl (hunk 3 above)
Comment (Zarathustra4) [reply src/main/resources/templates/menu.ftl]: fixed

**Review by justme1135 — APPROVED** [review]

**Pull request merged**